### ISSUE TITLE
[FR] Add sort options to search results, "explore" and "installed" tab

### ISSUE THREAD

**Issue by xlucn** [NeoApplications/Neo-Store#122]
v0.4.1

Now, when I change the sort options in "available" or "installed" tab, the other one will also change how it sorts the apps. Maybe it's better to let two tabs have their own independent sort order. I might not need it, but it will provide more choices for other users.

**Comment by machiav3lli** [comment]
To consider while working on #117

**Comment by oliverlew** [comment]
As neo-store redesigned its interface, this feature request should now become:

**Add sort options to search results, "explore" tab and "installed" tab.**

1. In search results, sometimes I want latest app on top. But the default sort priority is by relevance (first key words in appname or one-line description, then keywords in details).
2. If you add sort options to explore tab enabling sort by last update or app addition, that would replace the "latest" tab.
3. With sort options in "installed" tab, I can then see the app I recently installed, it's useful for managing apps or simply keeping track of what I just installed.
